### ISSUE TITLE
Api need so much

### ISSUE THREAD

**Issue by log901108** [AlUlkesh/sd_save_intermediate_images#34]
Does this extension provide api? I hope to know how to use it with using during using webui api.
Im trying to search how to use, but there is no info about it.
Thank you for your great work!!

**Comment by AlUlkesh** [comment]
No, unfortunately I haven't looked into a1111 api at all so far. Sorry.

**Comment by log901108** [comment]
sad. :(  thank you for your works though!

**Comment by pnavitha** [comment]
https://github.com/AlUlkesh/sd_save_intermediate_images is a very nice tool. But how to call it from the API when using Automatic1111 in the API mode? Plz help
